src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -16,18 +45,28 @@ export const TodoFilter = () => (
         type="text"
         className="input"
         placeholder="Search..."
+        value={query}
+        onChange={(event) => {
+          setQuery(event.target.value.toLowerCase());
+        }}
```

Comment:
The same. It is really important not to do this way in your tasks and future test tasks.
Check and fix it all over the code.